airflow/providers/google/suite/hooks/drive.py

```diff
@@ -218,3 +218,9 @@ def upload_file(self, local_location: str, remote_location: str) -> str:
         )
         self.log.info("File %s uploaded to gdrive://%s.", local_location, remote_location)
         return file.get("id")
+
+    def download_file(self, file_id: str, file_handle):
```

Comment:
A doctoring is required for every public function, otherwise pylons will fail on CI. I recommend installing pre-commits to check it locally:
https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#prerequisites-for-pre-commit-hooks